### PR TITLE
DHFPROD-7470: Toggle between graph view and tabular view

### PR DESCRIPTION
**Pull request by xnikhil08**
### Description
Just a dummy UI for graph view in Modeling tile. The list of entities are placed as a placeholder for now, which will be replaced by graph nodes in future.
The unit and integration tests are added in RTL.

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Code passes ESLint tests
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/pages/Modeling.tsx [hunk 1]
```diff
@@ -168,56 +169,109 @@ const Modeling: React.FC = () => {
     Revert All
   </MLButton>;
 
+  const handleViewChange = (view) => {
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui/src/pages/Modeling.tsx:172]: In handleViewChange, "graph" (or the config variable for it) should be the default so I would reverse the logic ("graph" as the fallback)?

File: marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <span>
+      <Dropdown
+        overlay={menu}
+        trigger={["click"]}
+        overlayClassName={styles.stepMenu}
+        placement="bottomRight"
+      >
+        <div className={styles.addButtonContainer}>
+          <MLButton aria-label="add-entity-type-relationship" size="default" type="primary">
+            Add{" "}
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui/src/components/modeling/graph-view/graph-view.tsx:53]: {" "} seems like a sketchy way to add space, it's not easily adjustable. Maybe wrap the "Add" in a tag and specify the right spacing via CSS? While we're at it, can we also clean up some of the indenting here (closing tags aligned)?

**Review by wooldridge — CHANGES_REQUESTED** [review]

File: marklogic-data-hub-central/ui/src/pages/Modeling.tsx (hunk 1 above)
Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/pages/Modeling.tsx:173]: Done.

File: marklogic-data-hub-central/ui/src/pages/Modeling.tsx [hunk 3]
```diff
@@ -20,11 +20,12 @@ import {MissingPagePermission} from "../config/messages.config";
 import {faLayerGroup, faKey} from "@fortawesome/free-solid-svg-icons";
 import arrayIcon from "../assets/icon_array.png";
 import relatedEntityIcon from "../assets/icon_related_entities.png";
+import GraphView from "../components/modeling/graph-view/graph-view";
 
 const Modeling: React.FC = () => {
   const {handleError} = useContext(UserContext);
   const {modelingOptions, setEntityTypeNamesArray, clearEntityModified} = useContext(ModelingContext);
-
+  let [view, setView] = useState("graph");
```
[the remark this answers is not in the document]
Comment (xnikhil08) [reply marklogic-data-hub-central/ui/src/pages/Modeling.tsx]: Done.

**Review by wooldridge — APPROVED** [review]

**Review by briantang — APPROVED** [review]

**Review by devinnicholson — APPROVED** [review]

**Pull request merged**